packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -59,10 +64,25 @@ describe('connectInfiniteHits', () => {
       const hits3 = [{}, {}, {}, {}, {}, {}, {}, {}];
 
       const res1 = getProvidedProps(null, null, {
-        results: { hits, page: 0, hitsPerPage: 6, nbPages: 10 },
+        results: {
+          hits,
+          page: 0,
+          hitsPerPage: 6,
+          nbPages: 10,
+          queryID: 'theQueryID_0',
+        },
       });
 
-      expect(res1.hits).toEqual(hits);
+      expect(res1.hits).toEqual(hits.map(hit => expect.objectContaining(hit)));
+      expect(res1.hits.map(hit => hit.__position)).toEqual([1, 2, 3, 4, 5, 6]);
```

Comment:
`_highlightResults` contains only one `_` but our `__position` contains two `__`. Is it on purpose?

Reply:
I had `_` initially in the RFC and someone mentioned that we only `__` is considered truly private
hence having `__hitIndex` currently, rather than `_hitIndex`.
Ready to change it if necessary (here and in vanilla)

Reply:
You're right it's probably better at least the API won't override it by mistake.